### DISCUSSION TITLE
Launchpad for YimMenu?

### DISCUSSION THREAD

**Discussion by ghost** [YimMenu/YimMenu#1718] [Ideas]
I think it would be nice if someone made an injector specifically for YimMenu with auto-update, changelog, etc.
I [did something similar](https://github.com/kbabrmover1995/YimMenu-Launchpad), as I see it, using Stand-Launchpad, remaking it for loading and injecting YimMenu.dll into the "AppData\Roaming\YimMenu\bin" folder and outputting the commit history and description as a changelog. Perhaps someone will be imbued with the idea of redesigning the interface so that it does not look so much like a stand-launchpad :) I'm just not very good at programming.
It seems to me that it would be much more convenient than constantly downloading a .dll file every day.

**Comment by Tayipc38** [comment]
Something tells me this is already a w.i.p

**Comment by ghost** [comment]
> Something tells me this is already a w.i.p

What do you mean by w.i.p?

**Comment by ghost** [comment]
Can I make a suggestion? 

Instead of making the launchpad locked to yimmenu (or stand, or any menu for that matter), can it be opened up to point at the menu's main download source, it's changelog, and it's folder with support for multiple menus? 

I already use stand's launchpad and co-load with yimmenu (since missions are fixed now). I'm not inclined to switch if it's just locked to another menu. But if you were to unlock it and make those "check for updates," "changelog," etc buttons customizeable with support for multiple menus then it would be much for versatile. I also like to experiment with other menus such as the tupoye fork of yimmenu. The launchpad would be far more valuable, futureproof, and versatile not being locked into a single menu, in my opinion.

**Reply by ghost** [reply]
I have added a standard dllList where you can add more mods besides YimMenu. But the implementation of loading from the link of the dll file, for example for the Stand, is different for each menu. For the Stand, you need to get the name of the latest version before generating the link itself. You can't do it the way it's done on the YimMenu website. Theoretically, I can later add an input for menus like TupoyeMenu so you can add your direct download links and co-load. If I understand you correctly.
I just wanted to offer the idea of ​​a convenient Launchpad specifically for the YimMenu. It's just that I personally got sick of the constant downloading of dll manually.

**Reply by ghost** [reply]
Now you can add an autoload cheat directly from the Internet, such as TupoyeMenu or some other by adding a download link. A new version will be loaded upon injection. Don't forget to set the delay value to 5
![image](https://github.com/YimMenu/YimMenu/assets/139077095/00ca2d6f-3fe7-4d8f-8d07-93683def6c5f)
